core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -212,6 +215,7 @@ public abstract class Store {
     protected final OperationChainValidator opChainValidator;
     private final SchemaOptimiser schemaOptimiser;
     private final Boolean addCoreOpHandlers;
+    private LocalDateTime createdTime = LocalDateTime.now();
```

Comment:
Do we want to initialise the time here if `Graph` is meant to use `store.setCreatedTime`

Reply:
ignore that
Do we want `createdTime` to be mutable? should we have a public `setCreatedTime` at all or should it be the responsibility of the store to set it's created time once when it is initialised

Reply:
You make a good point - I have simplified the `createdTime` setting to once the `Store` initialised and remove from the `Graph`